shared/oval/disable_interactive_boot.xml

```diff
@@ -1,34 +1,74 @@
 <def-group>
-  <definition class="compliance" id="disable_interactive_boot" version="2">
+  <definition class="compliance" id="disable_interactive_boot" version="3">
     <metadata>
-      <title>Disable Interactive Boot</title>
+      <title>Verify that Interactive Boot is Disabled</title>
       <affected family="unix">
         <platform>Red Hat Enterprise Linux 7</platform>
+        <platform>multi_platform_fedora</platform>
       </affected>
       <description>The ability for users to perform interactive startups should
       be disabled.</description>
-      <reference source="JL" ref_id="20140407" ref_url="test_attestation" />
+      <reference source="JL" ref_id="RHEL7_20160613" ref_url="test_attestation" />
+      <reference source="JL" ref_id="FEDORA23_20160613" ref_url="test_attestation" />
     </metadata>
-    <criteria>
-      <criterion test_ref="test_disable_interactive_boot" />
-      <!-- This needs to be enhanced yet to properly cover case of 'systemd.confirm_spawn=1'
-           use case on RHEL-7 and Fedora per:
-             https://rhsummit.files.wordpress.com/2014/04/summit_demystifying_systemd1.pdf Page #20
-           SSG Upstream ticket to come yet -->
+    <criteria operator="OR">
```

Comment:
@iankko shouldn't this be `"AND"` instead of `"OR"` because if you do not have a `GRUB_CMDLINE_LINUX_DEFAULT` line in `/etc/default/grub` but have `GRUB_CMDLINE_LINUX` with systemd.confirm_spawn=true this will evaluate as true rather than false, no?